pkg/apis/kops/cluster.go

```diff
@@ -276,6 +276,8 @@ type KopeioAuthenticationSpec struct {
 }
 
 type AwsAuthenticationSpec struct {
+	// Image is the AWS IAM Authenticator docker image to use
+	Image string `json:"imagew,omitempty"`
```

Comment:
@rifelpet Perhaps I have missed something, but is `imagew` a typo?

Reply:
@knight42 good catch! followup PRs coming shortly.